nominal/nominal.py

```diff
@@ -63,12 +71,20 @@ def set_token(base_url: str, token: str) -> None:
     _config.set_token(base_url, token)
 
 
+@typing_extensions.deprecated(
+    "nominal.get_default_client is deprecated and will be removed in a future version. "
+    "Use `nominal.NominalClient.from_profile` instead, see https://docs.nominal.io/python/profile-migration"
+)
 def get_default_client() -> NominalClient:
```

Comment:
can you move the impl of `get_default_client` to a private function without a deprecation warning, so that all of the funcs below which call `get_default_client` won't double (or triple, etc) warn?

Reply:
there may be other instances of this as well, haven't looked deeply